Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,336 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct SetUpLocationDrivenGeotriggersView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The popup to show the feature information.
+    @State private var popup: Popup?
+    
+    /// A Boolean value indicating whether to show the popup.
+    @State var isShowingPopup = false
+    
+    var body: some View {
+        MapView(map: model.map)
+            .locationDisplay(model.locationDisplay)
+            .task {
+                // Start geotrigger monitoring once the map loads.
+                do {
+                    try await model.map.load()
+                    model.startGeotriggerMonitoring()
+                } catch {
+                    model.error = error
+                }
+            }
+            .overlay(alignment: .top) {
+                // Status text overlay.
+                VStack {
+                    Text("\(model.fenceGeotriggerText)\n")
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                    
+                    Text(model.nearbyFeaturesText)
+                        .foregroundColor(.orange)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .padding(.top, -24)
```

Comment:
I'm confused by the manual negative padding here. 😵‍💫 A linebreak is added in the first Text, then a negative padding is used to put the second text closer to the top… Is it intended?

Occasionally manually adjust the padding to make the view look better is OK, but it is generally an anti-pattern for adaptive layout. Try to avoid them as much as we can.